linux/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+cd "$THIS_SCRIPT_PATH"
+
+builder_describe \
+  "Build Keyman for Linux." \
+  "clean" \
+  "configure" \
+  "build" \
+  "test" \
+  ":engine=ibus-keyman       ibus-keyman" \
+  "install                   install artifacts" \
+  "uninstall                 uninstall artifacts"
```

Comment:
```suggestion
  "Build Keyman for Linux." \
  ":engine=ibus-keyman       ibus-keyman" \
  "clean" \
  "configure" \
  "build" \
  "test" \
  "install                   install artifacts" \
  "uninstall                 uninstall artifacts"
```

Keep all targets and actions grouped?